hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/QuotaCache.java

```diff
@@ -353,39 +305,39 @@ public synchronized boolean triggerNow() {
       return super.triggerNow();
     }
 
+    /**
+     * Chore runs completely reset the state of the QuotaCache. This means that users get their full
+     * usage allotment back, since we throw away the accounting that was tracking their progress
+     * toward their limits. For this reason, it's recommended that chore runs should be infrequent.
+     */
```

Comment:
For us this is probably okay because we use per-second quotas, so the accounting loss isn't so significant, but it does seem like a big limitation for quotas with larger time units (which we do technically support, no idea how widely they're used). Anything we can do to maintain the usage accounting?

If it won't be easy, it could be worth an email to the dev list to see if anyone objects to this